web/src/main/java/com/es/phoneshop/web/controller/pages/OrderPageController.java

```diff
@@ -1,26 +1,65 @@
 package com.es.phoneshop.web.controller.pages;
 
-import com.es.core.service.OrderService;
 import com.es.core.exception.OutOfStockException;
+import com.es.core.model.order.Order;
+import com.es.core.service.OrderService;
+import com.es.phoneshop.web.controller.converter.OrderDtoConverter;
+import com.es.phoneshop.web.controller.dto.OrderDto;
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
 
 import javax.annotation.Resource;
+import javax.validation.Valid;
 
 @Controller
-@RequestMapping(value = "/order")
+@RequestMapping(value = "/orders")
 public class OrderPageController {
     @Resource
     private OrderService orderService;
 
-    @RequestMapping(method = RequestMethod.GET)
-    public void getOrder() throws OutOfStockException {
-        orderService.createOrder(null);
+    @ModelAttribute
+    public Order addOrder() {
+        return orderService.createOrder();
+    }
+
+    @ModelAttribute("OrderDto")
+    public OrderDto addOrderDto() {
+        return new OrderDto();
+    }
+
+    @GetMapping
+    public String getOrder() {
+        return "order";
     }
 
-    @RequestMapping(method = RequestMethod.POST)
-    public void placeOrder() throws OutOfStockException {
-        orderService.placeOrder(null);
+    @PostMapping
```

Comment:
Again urls, why not to make /orders/place POST endpoint to make it more readable and understandable